docs/admin/integration/weights-and-biases.md

```diff
@@ -0,0 +1,21 @@
+# Workspace Weights and Biases
+
+Weights and Biases is a commercial tool that provides experiment tracking, model visualization, and collaboration for machine learning projects. It helps researchers and developers keep track of their experiments, visualize their results, and compare different models in order to make informed decisions.
+
+When Wights and Biases are integrated into Run:ai Workspaces, researchers can easily create their custom work environments and have access to a toolbox of many research relevant tools in a single place. Reseachers can now create useful connectivity between the running Workspace and the relevant project using Weights Biases for experiment tracking.
+
+To configure Weights and Biases:
+
+1. Login to your account in [Weights and Biases](https://wandb.ai/site). If you do not have a valid account, you will need to create one.
```

Comment:
can you add {target=_blank} ? I have made it a standard that all external links are opened i a new tab.

Reply:
ok